cmd/courier/main.go

```diff
@@ -1,13 +1,17 @@
 package main
 
 import (
+	ulog "log"
+	"log/slog"
 	"os"
 	"os/signal"
+	goruntime "runtime"
```

Comment:
this is only necessary in mailroom because we have our own `runtime` struct

Reply:
reverted that